src/renderer/components/editorWithTabs/editor.vue

```diff
@@ -798,6 +798,7 @@ export default {
         }
       }
 
+      imageFolderPath = imageFolderPath.replace(/\${filename}/g, this.pathname)
```

Comment:
Please assign the replacement to a local variable and replace the usage below.

@bolshoytoster Whats the current status on this?

Reply:
@fxha it was already but I’ve added a commit that should do what you want.